test/runtests.jl

```diff
@@ -47,11 +47,11 @@ end
     typestr = ObjectiveC.encodetype(types...)
 
     @testset "simple" begin
-        function addone(x::T) where T
+        function addone(self, x::T) where T
             return x + one(T)
         end
         @assert sizeof(addone) == 0
-        block = Foundation.@objcblock(addone, Cint, (Cint,))
+        block = Foundation.@objcblock(addone, Cint, (Ptr{Object}, Cint,))
```

Comment:
```suggestion
        block = Foundation.@objcblock(addone, Cint, (id{Object}, Cint,))
```

Or is there a reason you're using `Ptr` instead of `id` here?